internal/pkg/cli/app_init.go

```diff
@@ -103,6 +111,14 @@ func (opts *InitAppOpts) Execute() error {
 		return fmt.Errorf("relative path of manifest file: %w", err)
 	}
 	opts.manifestPath = relPath
```

Comment:
nit: should we move all this manifest logic into a method? to make `Execute` easier to digest?
```
1. ensureNoExistingApp
2. createManifest
3. saveApp
🎉 success
```

Reply:
Delightful!